dual_net.py

```diff
@@ -541,10 +546,18 @@ def get_estimator():
 def _get_nontpu_estimator():
     session_config = tf.ConfigProto()
     session_config.gpu_options.allow_growth = True
-    run_config = tf.estimator.RunConfig(
-        save_summary_steps=FLAGS.summary_steps,
-        keep_checkpoint_max=FLAGS.keep_checkpoint_max,
-        session_config=session_config)
+    if FLAGS.num_gpus > 1:
```

Comment:
can you extract this num_gpus change to a separate PR?

Reply:
whups, sorry.  Sure can.

Reply:
(it doesn't work)